Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -695,9 +695,29 @@ def test_log
         system "git", "commit", "--allow-empty", "-m", "This is a test commit"
       end
     end
+
+    core_tap = CoreTap.new
+    formula_file = core_tap.formula_dir/"testball.rb"
+    formula_file.write <<-EOS.undent
+      class Testball < Formula
+        url "https://example.com/testball-0.1.tar.gz"
+      end
+    EOS
+
+    FileUtils.cd "#{core_tap.path}".chomp do
```

Comment:
> The `.chomp` doesn't look like it belongs here. Better yet, use the `Pathname#cd` method, i.e. `core_tap.path.cd do`, instead of `FileUtils.cd`.

Oops -- thanks for catching this. I was looking at [this line in `log.rb`](https://github.com/Homebrew/brew/blob/master/Library/Homebrew/cmd/log.rb#L22) and mistakenly applied the same logic to the return value of `core_tap.path`.

Just curious, what's the advantage of using `Pathname#cd` over `FileUtils.cd`? I see that `Pathname#cd` is part of the Homebrew DSL, but I'm still not sure why it might be preferred.